resources/assets/uad_lists.json

```
@@ -20941,5 +20941,14 @@
     "neededBy": [],
     "labels": [],
     "removal": "Expert"
+  },
+  {
+    "id": "com.asus.launcher",
+    "list": "Oem",
+    "description": "The launcher provided by Asus.\nWarning: disabling it causes the home gesture, task switcher gesture and task switcher button on the navbar to stop working, even when you are using a third party launcher",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
```

```suggestion
    "removal": "Expert"
```
I think it should be `Expert` instead. What do you think? See https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen.

---

Possibly I worded it slightly wrong: the issue is even when the third party launcher is selected disabling this package makes it impossible to use the task switcher (both by tapping the square button on the system navbar and by long-pull gesture when the gesture navigation is used) and to access the home screen at all when the gesture navigation is used (by short swiping up). I believe that should be considered the basic functionality and a severe degradation of everyday user experience.

---

> Possibly I worded it slightly wrong: the issue is even when the third party launcher is selected disabling this package makes it impossible to use the task switcher (both by tapping the square button on the system navbar and by long-pull gesture when the gesture navigation is used) and to access the home screen at all when the gesture navigation is used (by short swiping up). I believe that should be considered the basic functionality and a severe degradation of everyday user experience.

Yeah true, so you meant to say that you basically can't go back to the homescreen anymore at all times if you have this package disabled?

---

It seems using back gestures or back button will bring you to it *eventually* in most cases, but that depends on the app you have in foreground as it may just refuse to close by the use of back action

---

Yeah fair. Let's wait for @0x192 to take a look at this, eventually... Imo both standpoints are defendable.